how-to-guides/sequential-pipelines/scripts/model_validation.py

```diff
@@ -121,10 +122,12 @@
 
 # (Neptune) Log metrics to model registry
 run.wait()
-model_score = run["validation/metrics/scores/class_0"].fetch()
-model_version["model/metrics"] = model_score
+model_score = run["validation/metrics/scores"].fetch()
+model_version["metrics/validation/scores"] = model_score
 
 # (Neptune) Move model to stagging
 ACC_THRESHOLD = 0.50
```

Comment:
```suggestion
SCORE_THRESHOLD = 0.50
```

Lets rename this to `SCORE_THRESHOLD` as we are using the fbeta score, and not accuracy score